### ISSUE TITLE
No Targeting Below 80 m/z

### ISSUE THREAD

**Issue by mcbrider5002** [sdrogers/vimms#257]
ViMMS seems to have problems when scheduling fragmentation events below 80 m/z, even though defined constants would indicate the minimum should be 70 m/z. Fragmentation events between 70 and 80 m/z seem to be silently suppressed, though time does pass correctly between them. It's not currently clear at what stage this issue appears e.g. if the actual fragmentation event happens at all, or if it's a problem with writing to the .mzML, etc.

![image](https://user-images.githubusercontent.com/25082420/222237510-c922f81e-2b87-4456-814d-bdfcc6c3555c.png)

The above image shows a run of the intensity matching controller, which has many "unused" scans which must busy-wait, so they have been set to target some default value. In this case the default value is 100 m/z, so a line of fragmentation events can be clearly observed at 100 m/z.

![image](https://user-images.githubusercontent.com/25082420/222237833-6c455589-62de-41ce-8446-dbb7808df629.png)

This image shows the same run, but with the default target set to 75 m/z. No line of fragmentation events can be observed here, suggesting that these scans are dropped somehow.

![image](https://user-images.githubusercontent.com/25082420/222237983-37d4aeb2-c90f-4e24-ae01-48f384b84046.png)
![image](https://user-images.githubusercontent.com/25082420/222238019-cba7bb65-1636-4e76-bbc5-e900bda9fdcf.png)

In these images we see our picked peak boxes, i.e. our targets for the evaluation. Blue are fragmented: red unfragmented. At the bottom we see a clump of red (unfragmented) boxes below 80 m/z. This suggests that these cannot be targeted to cover them. The two controllers here are Intensity Non-Overlap and the intensity matching controller, which have little in common in their specific implementations, which suggests this may be a broader issue.

**Comment by joewandy** [comment]
is this still a problem @mcbrider5002 ?

**Comment by mcbrider5002** [comment]
Yes, it is. My workaround for it is that I've changed the MZMine parameters to not pick boxes below 80 m/z. But it still happens, and we don't know why it happens so it could cause us problems in future.

**Comment by joewandy** [comment]
Resolved by merged PR #329. The default MS2 product-ion scan range no longer has the hard-coded 70 m/z lower bound, and users can still restore/customize the range via AdvancedParams(default_ms2_scan_window=...).
